src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -442,10 +443,10 @@ var adyenHelperObj = {
     }
 
     const shopperDetails = {
-      firstName: args.order.getBillingAddress().firstName,
+      firstName: args.order.getDefaultShipment().getShippingAddress().firstName,
```

Comment:
Why updating names to Shipping instead of Billing?

Reply:
We need these details when creating a session object. At that stage we do not yet have any Billing information. That's why I changed it to shipping. 
Should we introduce a separate function for the session shopper details?
I guess it does make sense for this to be Billing.

Reply:
I think it would make sense to have the Billing details as shopper details rather than Shipping.
Can we update the shopper details after the Session is created?

Reply:
I'll change it so Shipping is used as a Fallback if Billing is not available. This way we use shipping to create the session, but we'll use billing for payments